### ISSUE TITLE
SteamVR_LoadLevel.cs overlay sorting issue

### ISSUE THREAD

**Issue by Blosserdw** [ValveSoftware/steamvr_unity_plugin#41]
**Problem:**
In the SteamVR_LoadLevel.cs script, the loadingScreen texture overlay seems to always sort over top of the progressBarEmpty/progressBarFull texture overlay. I do see the progress image fade in, then fade out immediately after. Loading takes place then, when the scene is loaded, it shows up again for a split second before I'm dumped into the loaded scene.

**My Solution:**
I solved this by writing
`overlay.SetOverlaySortOrder(handle, orderNum);`
in the GetOverlayHandle() function next to where it's setting the alpha, incrementing the orderNum so each call would put the subsequent overlays on top of the one before it. Seems to show up fine after adding that.

Here are the properties I've set up on the script in case it somehow turns out that I'm the source of my own problem :)
![Screenshot](http://i.imgur.com/8yXNNR8.png)

Hope that helps!

**Comment by aleiby** [comment]
Overlays should be sorted by position.  Is that not working?  Or do you have them co-located?

**Comment by Blosserdw** [comment]
I did not set transforms for them in the inspector, so they both default to the same location (script object's transform). 

The script says that setting the transforms for these overlays is optional, which is why I figured it was an issue that they weren't displaying properly when they're at the same location. Although maybe that's just due to my misunderstanding of how the overlays work.

`// Optional overrides for where to display loading screen and progress bar overlays.`
`// Otherwise defaults to using this object's transform.`
`public Transform loadingScreenTransform, progressBarTransform;`
